hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -5242,7 +5242,7 @@ objects:
                 restartPolicy: Never
                 containers:
                 - name: osd-logging-freeze
-                  image: quay.io/openshift/origin-cli:4.7.0
```

Comment:
Is this a regeneration error? I don't see a matching change for this

Reply:
Oh, looks like 1d7331cddb7881a8a8bd592467cf1edece9350dc

Reply:
That's puzzled me too - I'm not sure why it's changing that image URL. It happens when I run `make` straight off master as it currently exists, too.